### PR TITLE
add explosive devices to missing units, close #4181

### LINKED ISSUE #4181: Explosive Specialists of Viper and CTRG, do not have an activator M57
**Arma 3 Version:** `1.62` (stable)
**CBA Version:** `3.0.0.160713` (stable)
**ACE3 Version:** `3.6.2` (stable)

**Mods:**
- `@CBA_A3`
- `@ace`

**Description:**
These explosive specialists , do not have an activator M57(or another) of explosives. For this reason, can not use a distant detonation of explosives (Claymor, SLAM, C4 etc)

**Steps to reproduce:**
Play for explosive specialists of this faction

**Where did the issue occur?**
Editor (Singleplayer) / Editor (Multiplayer) 

**Placed Modules:**
Only Advanced and Basic Medical Modules

**RPT log file:**
not needed

### PR DESCRIPTION
**Pull request by commy2**
**When merged this pull request will:**
- fix #4181
- also formating

### CODE REVIEW

File: addons/explosives/CfgVehicles.hpp [hunk 1]
```diff
@@ -279,38 +279,41 @@ class CfgVehicles {
 
     // Add ACE_Clacker and ACE_DefusalKit to every explosive unit.
     #define MACRO_ADDMINEKIT \
-    items[] = {"FirstAidKit", "ACE_Clacker", "ACE_DefusalKit"}; \
-    respawnitems[] = {"FirstAidKit", "ACE_Clacker", "ACE_DefusalKit"};
+    items[] = {"FirstAidKit","ACE_Clacker","ACE_DefusalKit"}; \
+    respawnitems[] = {"FirstAidKit","ACE_Clacker","ACE_DefusalKit"};
```
Comment (jonpas) [inline addons/explosives/CfgVehicles.hpp]: Unnecessary changes, unless we decide on spacing standard. Changes like that just become merge conflicts.

Comment (commy2) [inline addons/explosives/CfgVehicles.hpp]: bla bla bla
If you don't want merge conflicts, merge PRs immediately

Comment (jonpas) [inline addons/explosives/CfgVehicles.hpp]: That's a terrible strategy.
